sos/policies/distros/__init__.py

```diff
@@ -900,31 +894,24 @@ def upload_ftp(self, url=None, directory=None, user=None, password=None):
                 raise Exception("connection failed, did you set a user and "
                                 "password?")
             session.cwd(directory)
-        except socket.timeout:
-            raise Exception(f"timeout hit while connecting to {url}")
-        except socket.gaierror:
-            raise Exception(f"unable to connect to {url}")
```

Comment:
Same here, why are we dropping these handlers and the messaging that is more informative for end users?

Reply:
The `url` is displayed by the default exceptions, so think that's a useful info - restored these two.